docs/native-docker.md

```diff
@@ -142,7 +142,9 @@ and defaults to `0`.  Like `hostPort`, If the value is `0`, a random port will
 be assigned.  The values for random service ports are in the
 range `[local_port_min, local_port_max]` where `local_port_min` and
 `local_port_max` are command line options with default values of `10000` and
-`20000`, respectively.
+`20000`, respectively. Note that HAProxy supports forwarding only TCP traffic.
+If the `"protocol"` parameter is set to `"udp"`, HAProxy will forward only
```

Comment:
Maybe reference the haproxy-marathon-bridge in your description. It's only a sample script how you could do service discovery/load balancing and by no means the canonical way of how to do service discovery in Marathon.